doc/reference/tokenratelimitpolicy.md

```diff
@@ -0,0 +1,213 @@
+# TokenRateLimitPolicy
```

Comment:
Disclosure: this is largely generated (seems to have done a decent job). Needs a human sanity check.